Examples/JamitFoundationExample/App/Sources/ListView/SampleListViewController.swift

```diff
@@ -0,0 +1,88 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import JamitFoundation
+import UIKit
+
+final class SampleListViewController: ListViewController<ListItemView> {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        title = "ListViewController"
+
+        model = ListViewModel.init(
+            height: .constant(60),
+            items: [
+                ListItemViewModel(title: "TableView"),
+                ListItemViewModel(title: "ActionView"),
+                ListItemViewModel(title: "ScrollView")
+            ]
```

Comment:
You can define all titles in a variable before, map to `ListItemViewModel`, and assign it as items here.

For example:
```
let itemModels = ["A", "B", "C"].map { ListItemViewModel(title: $0) }
model = .init(height: .constant(60), items: itemModels)
```

Reply:
Done.